README.md

```diff
@@ -86,52 +86,75 @@ KBC_TEST_ADMIN_TOKEN=token_of_another_admin_having_mfa_disabled
 KBC_TEST_ADMIN_WITH_MFA_EMAIL=email_of_another_admin_having_mfa_enabled
 KBC_TEST_ADMIN_WITH_MFA_TOKEN=token_of_another_admin_having_mfa_enabled
 
-# OPTIONAL - required only for running file storage test (tests are skipped by default)
-TEST_ABS_ACCOUNT_KEY=
-TEST_ABS_ACCOUNT_NAME=
-TEST_ABS_CONTAINER_NAME=
-TEST_ABS_REGION=
-TEST_ABS_ROTATE_ACCOUNT_KEY=
-TEST_S3_ROTATE_KEY=
-TEST_S3_ROTATE_SECRET=
-TEST_S3_FILES_BUCKET=
-TEST_S3_KEY=
-TEST_S3_REGION=
-TEST_S3_SECRET=
-TEST_GCS_KEYFILE_JSON=
-TEST_GCS_KEYFILE_ROTATE_JSON=
-TEST_GCS_FILES_BUCKET=
-TEST_GCS_REGION=
-
 # OPTIONAL - required only for running testCreateStorageBackend, you have to have new snowflake backend and fill credentials into following environment variables
-
 KBC_TEST_SNOWFLAKE_BACKEND_NAME=
 KBC_TEST_SNOWFLAKE_BACKEND_PASSWORD=
 KBC_TEST_SNOWFLAKE_HOST=
 KBC_TEST_SNOWFLAKE_WAREHOUSE=
 KBC_TEST_SNOWFLAKE_BACKEND_REGION=
-
 ```
 
-Source newly created file and run tests:
-
+Run tests
 ```bash
 docker-compose run --rm dev composer tests
 ```
 
-### Required variables
```

Comment:
tady to popsani variables by nemelo zmizet. asi bych ho mergnul s tim co je na lajne 75. To vysvetleni je tam dulezite

Reply:
jo, jasne, to je chyba, prehledl jsem